keel-core/src/main/kotlin/com/netflix/spinnaker/keel/exceptions/InvalidRegexException.kt

```diff
@@ -31,4 +31,4 @@ class InvalidRegexException(
 class NoDockerImageSatisfiesConstraints(
   val artifactName: String,
   val environment: String
-) : ResourceCurrentlyUnresolvable("No docker image found for artifact $artifactName in $environment")
+) : ResourceCurrentlyUnresolvable("No docker image found for artifact $artifactName that satisfies constraints in environment $environment")
```

Comment:
Use the same language as for AMIs to make clear the problem is not that an image was not found, but that no image that satisfies constraints was found.
https://github.com/spinnaker/keel/blob/08289eb264e33538c8630003d059343504843845/keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/NoImageFound.kt#L28-L29